drivers/sensor/openrocket/openrocket_sensors.c

```diff
@@ -102,10 +117,10 @@ static void or_event_thread_handler(void) {
         int time_to_wait_ms = (int) ((or_events[i].time_s - time) * 1000);
         k_msleep(time_to_wait_ms);
         time = or_events[i].time_s;
-        LOG_INF("OpenRocket event %s at time T+%.3f", event_to_str(or_events[i].event), or_events[i].time_s);
+        printk("OpenRocket event %s at time T+%.3f\n", event_to_str(or_events[i].event), (double) or_events[i].time_s);
         i++;
     }
-    LOG_INF("OpenRocket flight over");
+    printk("OpenRocket flight over\n");
```

Comment:
Why no log?

Reply:
It made the event log kinda weird to use cuz you would enable it but it could be mystery disabled by the log subsystem and its annoying to hunt down when its not working

Reply:
Mm should probably investigate why. I guess it might be acceptable in this case, if its just simulation, but I don't recall having issues with it

Reply:
Its not so much an issue of the log system not doing what its told. its that you could enable the openrocket log kconfig but the openrocket_log_level couldnt be suffeciently high enough to print it out. so it would look like its turned on but it really wasn't. 

I couldnt get kconfig to do a select on a choice like log level. ik you can have a configuration select another boolean on but i could not figure out how to say use log level inf if this one is selected